### ISSUE TITLE
Enemy ccminer with x17 algo: Stratum authentication failed

### ISSUE THREAD

**Issue by borzaka** [Sniffdog/Sniffdogminer#199]
### Command:
`Bin\NVIDIA-Enemy\ccminer.exe -a x17 -o stratum+tcp://x17.mine.ahashpool.com:3737 -u 17nWUDPvTfMtCwbPQs98NaKd2J559paNo1 -p ID=nvidiaworker,c=BTC`

![image](https://user-images.githubusercontent.com/1230402/38270596-26207e04-3784-11e8-84c7-ad7bc2a2d37e.png)

No success for the 15th times. Sniffdogminer will stop trying and kick that miner?

### Basic info:
- `startsniffin_Ahashpool_Bitcoin.bat`
Modified only `-Workername`, `-RigName` and `-Wallet`
- 3x 1080 Ti
- 3x 1070 Ti

**Comment by Sniffdog** [comment]
Ill check into it

**Comment by Sniffdog** [comment]
had to place x17 into ccminer Alexis for now and updated enemy miner...

You will have to do some manual steps to receive enemy miners from sniffdog..... the zip files and links are in the Bin Folders..... Also, you will need to turn on x16r and x16s in Miners folders on enemy's files...

Right now nevermore is default in Sniffdog for raven and pigeon.... until I can get these enemy miners to unpack properly from MEGA

**Comment by sephir69** [comment]
ocminer suprminer should be added as well nice speed and no fee

**Comment by Sniffdog** [comment]
I will have a look at it... thanks for the input...

**Comment by Sniffdog** [comment]
ok hashrate is better... that is important

Thank you for showing me this

**Comment by sephir69** [comment]
yeah no fee isn't a big deal but the speed on suprminer is nice... I meant more for x16r though didn't realize this was the x17 thread :P

**Comment by Sniffdog** [comment]
haha all good... Suprminer is in new release
